contracts/extensions/VotingReputation.sol

```diff
@@ -314,29 +322,7 @@ contract VotingReputation is ColonyExtension, PatriciaTreeProofs {
   )
     public
   {
-    createMotion(_domainId, _childSkillIndex, _altTarget, _action, _key, _value, _branchMask, _siblings);
-  }
-
-  /// @notice Create a motion in any domain (DEPRECATED)
-  /// @param _domainId The domain where we vote on the motion
-  /// @param _childSkillIndex The childSkillIndex pointing to the domain of the action
-  /// @param _action A bytes array encoding a function call
-  /// @param _key Reputation tree key for the domain
-  /// @param _value Reputation tree value for the domain
-  /// @param _branchMask The branchmask of the proof
-  /// @param _siblings The siblings of the proof
-  function createDomainMotion(
-    uint256 _domainId,
-    uint256 _childSkillIndex,
-    bytes memory _action,
-    bytes memory _key,
-    bytes memory _value,
-    uint256 _branchMask,
-    bytes32[] memory _siblings
-  )
-    public
-  {
-    createDomainMotion(_domainId, _childSkillIndex, address(0x0), _action, _key, _value, _branchMask, _siblings);
+    createMotion(_domainId, _childSkillIndex, address(0x0), _action, _key, _value, _branchMask, _siblings);
```

Comment:
Didn't we deprecate domain motions in #949  ?

Reply:
Yes? I'm not sure what you're saying here :disappointed:

Reply:
Oh rather, didn't we add support for `altTargets`? Why is this PR changing the interface?

Reply:
Oh, I see. The version of `createDomainMotion` that took `altTarget` was both added and deprecated in that PR, which shouldn't have happened. I'm removing it so it's never released.